### ISSUE TITLE
LookupClient does not handle numbers with spaces in them

### ISSUE THREAD

**Issue by look** [twilio/twilio-ruby#153]
It looks like phone numbers passed to the lookup client have to be URL encoded. This is not reflected in the [quickstart for the PHP client](https://www.twilio.com/docs/quickstart/php/lookups), the VOIP example on the [product page](https://www.twilio.com/lookup), or mentioned in the [Ruby client documentation](https://www.twilio.com/blog/2015/04/validate-phone-numbers-in-ruby-using-the-lookup-api.html). Also, failure to encode the input causes an unhandled exception.

Examples:

```
lookup_client.phone_numbers.get("+13123131434").phone_number
=> "+13123131434"

lookup_client.phone_numbers.get("+1 312 313 1434").phone_number
NoMethodError: undefined method `[]' for nil:NilClass
	from /Users/lfrancl/.rvm/gems/ruby-2.0.0-p353/gems/twilio-ruby-3.16.1/lib/twilio-ruby/rest/base_client.rb:134:in `connect_and_send'
	from /Users/lfrancl/.rvm/gems/ruby-2.0.0-p353/gems/twilio-ruby-3.16.1/lib/twilio-ruby/rest/base_client.rb:60:in `block (2 levels) in <class:BaseClient>'
	from /Users/lfrancl/.rvm/gems/ruby-2.0.0-p353/gems/twilio-ruby-3.16.1/lib/twilio-ruby/rest/instance_resource.rb:69:in `method_missing'
	from (irb):48
	from /Users/lfrancl/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.21/lib/rails/commands/console.rb:47:in `start'
	from /Users/lfrancl/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.21/lib/rails/commands/console.rb:8:in `start'
	from /Users/lfrancl/.rvm/gems/ruby-2.0.0-p353/gems/railties-3.2.21/lib/rails/commands.rb:41:in `<top (required)>'
	from script/rails:6:in `require'
	from script/rails:6:in `<main>'

lookup_client.phone_numbers.get(URI.encode("+1 312 313 1434")).phone_number
=> "+13123131434"
```

**Comment by philnash** [comment]
Good news. The unhandled exception has been sorted for a future version in PR #149. And I just opened #154 to deal with this.

I will also get someone to look at the documentation. Thanks for reporting!

**Comment by philnash** [comment]
I have just released version 4.2.1 of the ruby gem that will URI encode phone numbers when calling this method now. Thanks for reporting!
